units/user-config.target

```diff
@@ -11,3 +11,9 @@ After=user-cloudinit@var-lib-coreos\x2dinstall-user_data.path
 
 Requires=user-cloudinit-proc-cmdline.service
 After=user-cloudinit-proc-cmdline.service
+
+Requires=user-config-vmware-tools.service
```

Comment:
Why is this service being started? The [VMware OEM config](https://github.com/coreos/coreos-overlay/blob/fe61d781d312b35f7cc3b6890b85c826e48fa5a5/coreos-base/oem-vmware/files/cloud-config.yml#L27) will start this directly.